images/tf_operator/build_and_push.py

```diff
@@ -112,6 +118,8 @@ def run(command, cwd=None):
   else:
     run(["docker", "build", "-t", image,  context_dir])
     logging.info("Built image: %s", image)
+  
+  if args.should_push:
```

Comment:
The if args.should_push statement should be inside the else block. We only push the image if we aren't using GCB.